app.js

```diff
@@ -400,11 +401,20 @@ var eventsModule = function () {
                 });
         }
 
+        var sumFrameDuration = function (efFrames) {
+            var frameSum = 0;
+            for (var i = 0; i < efFrames.length; i++) {
+                frameSum += ((efFrames[i].ef.EndTime - efFrames[i].ef.StartTime) / 1000 / 60);
+            }
+            return frameSum;
+        }
+
```

Comment:
Now i'll need to look at the rest of that function and see if i can perform similar refactorings! i'm sure my code can use the same simplifications.